src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.csproj

```diff
@@ -19,6 +20,7 @@
     <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsUnix)' == 'true'">true</GeneratePlatformNotSupportedAssembly>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetGroup)' == 'netcore50'">
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <PackageTargetFramework>netstandard1.4</PackageTargetFramework>
```

Comment:
If you're going to package as netstandard1.4 you should probably use the netstandard1.4 TargetGroup instead of netcore50.